x/evm/keeper/state_transition.go

```diff
@@ -210,32 +210,22 @@ func (k *Keeper) ApplyTransaction(ctx sdk.Context, msgEth *types.MsgEthereumTx)
 		bloomReceipt = ethtypes.BytesToBloom(bloom.Bytes())
 	}
 
-	cumulativeGasUsed := res.GasUsed
```

Comment:
seems we also dont use cumulativeGasUsed in [PostTxProcessing](https://github.com/crypto-org-chain/cronos/blob/main/x/cronos/keeper/evm_hooks.go#L48)

Reply:
yeah, I bet no one would need this in consensus.